tests/integration/__init__.py

```diff
@@ -565,6 +542,12 @@ def use_cluster(cluster_name, nodes, ipformat=None, start=True, workloads=None,
                     CCM_CLUSTER.set_configuration_options({'enable_user_defined_functions': True})
                     if Version(cassandra_version) >= Version('3.0'):
                         CCM_CLUSTER.set_configuration_options({'enable_scripted_user_defined_functions': True})
+                        if Version(cassandra_version) >= Version('4.0-a'):
+                            CCM_CLUSTER.set_configuration_options({
+                                'enable_materialized_views': True,
+                                'enable_sasi_indexes': True,
+                                'enable_transient_replication': True,
```

Comment:
I'm assuming these are just guardrail settings and a "don't let you shoot yourself in the foot unless you explicitly agree to it" type of thing. Is there any downside to leaving them on globally? Since we need to support these features, I figure there's no harm in turning them on globally unless we need to check the way something works with these features disabled.

Reply:
yeah, sounds good! thanks